src/entities/Customer.java

```diff
@@ -27,7 +27,7 @@ public Customer(String name, int age, String username, String password, String e
     }
 
 
-    public boolean login(ArrayList<Customer> customerDB) throws IOException {
+    public boolean login(ArrayList<Object> customerDB) throws IOException {
```

Comment:
Since in our interface, we changed the ArrayList to take in Object, we have to update it here as well.